telegram/ext/_applicationbuilder.py

```diff
@@ -564,15 +564,31 @@ def pool_timeout(self: BuilderType, pool_timeout: Optional[float]) -> BuilderTyp
     def http_version(self: BuilderType, http_version: str) -> BuilderType:
         """Sets the HTTP protocol version which is used for the
         :paramref:`~telegram.request.HTTPXRequest.http_version` parameter of
-        :attr:`telegram.Bot.request`. By default, HTTP/2 is used.
+        :attr:`telegram.Bot.request`. By default, HTTP/1.1 is used.
 
         .. seealso:: :meth:`get_updates_http_version`
 
         .. versionadded:: 20.1
+        .. versionchanged:: 20.1.1
+            Reset the default version to 1.1.
+
+        Note:
+            HTTPX currently errors out with HTTP/2 when Telegram cancels the keepalive connection,
```

Comment:
The phrasing is somewhat awkward IMHO. I made a suggestion at https://github.com/python-telegram-bot/python-telegram-bot/pull/3576#discussion_r1126930937